config/environments/production.rb

```diff
@@ -3,6 +3,7 @@
   config.webpacker.check_yarn_integrity = false
 
   config.cache_classes = true
+  config.cache_store = :redis_cache_store, { url: IdentityConfig.store.redis_url }
```

Comment:
maybe share this with development too?

Reply:
I'm not sure I guess, I was thinking no, but I'm open to adding it.